l10n_br_base/__manifest__.py

```diff
@@ -1,41 +1,106 @@
 # -*- coding: utf-8 -*-
+#
 # Copyright (C) 2009 - TODAY Renato Lima - Akretion
+# Copyright 2017 Taŭga Tecnologia
+#   Aristides Caldeira <aristides.caldeira@tauga.com.br>
+# Copyright 2017 KMEE
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+#
 
 {
     'name': 'Brazilian Localization Base',
     'category': 'Localisation',
     'license': 'AGPL-3',
-    'author': 'Akretion, Odoo Community Association (OCA)',
+    'author': 'Odoo Community Association (OCA), Akretion',
     'website': 'http://odoo-brasil.org',
     'version': '10.0.1.0.0',
     'depends': [
         'base',
+        'mail',
         'base_setup',
+        'decimal_precision',
+        'product',
```

Comment:
l10n_br_base should not depend on the product module, the code related to product should go in the  l10n_br_product module instead